pallets/loans/src/entities/pricing/external.rs

```diff
@@ -126,7 +126,7 @@ impl<T: Config> ExternalActivePricing<T> {
 		Ok(T::PriceRegistry::get(&self.info.price_id)?.1)
 	}
 
-	pub fn current_interest(&self) -> Result<T::Balance, DispatchError> {
+	pub fn interest_accrued(&self) -> Result<T::Balance, DispatchError> {
```

Comment:
I've changed this name because `current_interest` seems to return something like "2%",  instead of an interest accrued during the time.

Reply:
No longer apply